README.md

```diff
@@ -1,93 +1,27 @@
-*Looking for a shareable component template? Go here --> [sveltejs/component-template](https://github.com/sveltejs/component-template)*
+# Big Brother Manager
 
----
+A front-end interface to bb-bot api. 
 
-# svelte app
+# How to Run locally
 
-This is a project template for [Svelte](https://svelte.dev) apps. It lives at https://github.com/sveltejs/template.
+1. Use docker-compose up to start up the stack needed.
```

Comment:
@gutorc92, put here the code needed to startup the stack with the provided docker-compose.yml